nextest-runner/src/runner/executor.rs

```diff
@@ -210,11 +210,25 @@ impl<'a> ExecutorContext<'a> {
 
         let (req_rx_tx, req_rx_rx) = oneshot::channel();
 
+        // Compute the command line for this test.
+        let ctx = TestExecuteContext {
+            profile_name: self.profile.name(),
+            double_spawn: &self.double_spawn,
+            target_runner: &self.target_runner,
+        };
+        let command_line = test.instance.command_line(
+            &ctx,
+            self.test_list,
+            settings.run_wrapper(),
+            settings.run_extra_args(),
+        );
```

Comment:
Hmm it looks like two separate `TestExecuteContext`s are being constructed now -- one here and one inside `run_test_inner`. This kind of pattern can lead to issues in case the two fall out of sync for some reason. Please collapse them down into one context creation.